dbt/task/compiler/linker.py

```diff
@@ -0,0 +1,93 @@
+import sqlparse
```

Comment:
the choice of namespace here is confusing - it's under the `compiler` namespace, but it's used by the `run` task?  I recommend just moving this into `run.py` for now